src/ConcreteOperations/interior.jl

```diff
@@ -0,0 +1,28 @@
+export is_interior_point
+
+
+"""
+    is_interior_point(d::AbstractVector, P::LazySet; p=Inf, ε=_TOL_F64.rtol)
+
+Check if the point `d` is contained in the interior of the convex set `P`.
+
+### Input
+
+- `d`  -- coordinate of point
+- `P`  -- lazy set
+- `p`  -- norm of the Ball used to apply the error tolerance
+- `ε`  -- error tolerance of check
+
+### Output
+
+Boolean which indicates if point `d` is contained in `P`.
+
+### Algorithm
+
+The implementation checks if a `BallInf` with center `d` and radius `ε` is
+contained in the set `P` which results in a numerical check for `d ∈ interior(P)`
+with error tolerance `ε` with default value `_TOL_F64.rtol`.
+"""
+function is_interior_point(d::AbstractVector, P::LazySet; p=Inf, ε=_TOL_F64.rtol)
```

Comment:
I think you should make the default for `ε` depend on the numeric type in `d`/`P`. 
```suggestion
function is_interior_point(d::AbstractVector{N}, P::LazySet{N}; p=Inf, ε=_rtol(N)) where {N<:Real}
```

Reply:
I am also not sure whether this parameter should be passed by the user or whether the user should rather modify `_rtol`.

Reply:
I would keep this, as both are good IMO: either pass it for single-use or modify it globally for longer scripts or consistency through a given application that uses different functions that rely on `comparisons.jl`.

Reply:
(Taking this change of course implies that you adapt the docstring and the reference in `docs/src/lib/operations.md`.)